### ISSUE TITLE
Ruby updates

### ISSUE THREAD

**Issue by TheTechRobo** [Chris911/iStats#82]
I'm embarrassed about this, but I still use Mac OS  X Lion... I just **really** like Aqua. 
There's one problem... isn't the only way to update Ruby is with Homebrew? 
Homebrew drops support for out-of-date OSes... like all the versions of Mac OS I have at home :)
Whenever I try to install, it shows this:  
`Anfunfun's-MacBook-Air:~ computer$ gem install iStats
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions into the /Library/Ruby/Gems/1.8 directory.
Anfunfuns-MacBook-Air:~ computer$ sudo ARCHFLAGS=-Wno-error=unused-command-line-argument-hard-error-in-future gem install iStats
Password:
ERROR:  Error installing iStats:
	iStats requires Ruby version >= 1.9.3.`

So in other words I'm stuck? Homebrew said it wouldn't have any support on Yosemite, and the install failed for.. even installing Homebrew on Yosemite. Doubt it would work on an even older OS. 

MacBook Air (October 2010)
OS X Lion 10.7.5
Build 11G63

Thanks!

**Comment by Chris911** [comment]
Try with more permissions: `sudo gem install iStats`

**Comment by TheTechRobo** [comment]
Okay, trying that...

**Comment by TheTechRobo** [comment]
Last login: Wed Jul 10 11:09:53 on ttys000
Anfunfuns-MacBook-Air:~ computer$ su moralater9
Password:
bash-3.2$ sudo gem install iStats

WARNING: Improper use of the sudo command could lead to data loss
or the deletion of important system files. Please double-check your
typing when using sudo. Type "man sudo" for more information.

To proceed, enter your password, or type Ctrl-C to abort.

Password:
ERROR:  Error installing iStats:
	iStats requires Ruby version >= 1.9.3.
bash-3.2$

**Comment by prantlf** [comment]
Alternatively, you can avoid `sudo` by instalklig `ruby` to your home directory.

Install `rbenv` using `brew`. Install and enable any version of `ruby`, like `2.6.3`. Installed versions are located in `~/.rbenv`. If you install gems globally like `gem install iStats`, they will not need `sudo`, because they will be written to your home directory.
